mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/UserScreen.kt

```diff
@@ -30,11 +25,10 @@ fun UserScreen(
     header: @Composable () -> Unit,
     profileTabBar: @Composable () -> Unit,
     matches: List<ChessMatch>,
+    onGameItemClick: (ChessMatch) -> Unit,

```

Comment:
Maybe `onMatchClick` would be a more consistent naming with our existing `matches` parameter.